openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -193,7 +196,7 @@ def _email_url_parser(url_name, extra_param=None):
     site_name = configuration_helpers.get_value('SITE_NAME', settings.SITE_NAME)
     dashboard_url_path = reverse(url_name) + extra_param if extra_param else reverse(url_name)
     dashboard_link_parts = ("https", site_name, dashboard_url_path, '', '', '')
-    return urlparse.urlunparse(dashboard_link_parts)
+    return six.moves.urllib.parse.urlunparse(dashboard_link_parts)  # pylint: disable=too-many-function-args
```

Comment:
Any idea why it started throwing that warning with only one argument?  That's a little concerning.

Reply:
It looks like it may be fixed in a more recent version of `asteroid` but we aren't quite on it yet: https://github.com/PyCQA/pylint/issues/1965